xwalk_android.gypi

```diff
@@ -213,6 +213,101 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+        'libxwalkcore_so': '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+        'classes_dex': '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+        'icudtl_dat': '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+        'xwalk_pak':'<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+      },
+      'actions': [
```

Comment:
What if instead of having a series of hardcoded actions like this you just have one that takes a list of arguments (files)? The same Python script could also be used in a mode that just outputs the file names with ".lzma" appended to them, sort of like `src/build/grit_action.gypi`.